assets/components/featuredProductHero/featuredProductHero.jsx

```diff
@@ -37,7 +35,7 @@ export default function FeaturedProductHero(props: PropTypes) {
     product,
   } = props;
 
-  const timerClassName = classNameWithModifiers('component-featured-product-hero__countdownbox', hasTimer ? [] : ['hidden']);
```

Comment:
looking at this now perhaps it would be cleaner to pass `product === 'DigitalPack'` as `hasTimer` higher down the line?

Reply:
hasTimer isn't in use anywhere. So it doesn't make sense to make a prop (says Akash) :)

Reply:
it's not anywhere because the timer wasn't showing until this PR :D the idea behind it was to enable/disable the timer from outside the component, just to make the component a bit more reusable